testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -321,6 +333,8 @@ else if (getHiveVersionMajor() == 3 && isFormat.test("orc")) {
                         0.5,
                         -1.5))
                 .put("double_to_float", ImmutableList.of(0.5, -1.5))
+                .put("double_to_string", Arrays.asList("12345.12345", coercedNaN))
```

Comment:
ImmutableList too?

Reply:
`coercedNaN` can be null - so ImmutableList might fails.

Reply:
Ah, got it.